site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -859,6 +859,7 @@ public function showDetails($gradeable_id, $sort = "id", $direction = "ASC") {
             $this->core->redirect($this->core->buildCourseUrl());
         }
         $anon_mode = isset($_COOKIE['anon_mode']) && $_COOKIE['anon_mode'] === 'on';
+        $sort = isset($_COOKIE['sort']) ? $_COOKIE['sort'] : "id";
```

Comment:
Just for consistency with the other strings on this line:
```suggestion
        $sort = isset($_COOKIE['sort']) ? $_COOKIE['sort'] : 'id';
```